app/models/stage.rb

```diff
@@ -149,7 +149,10 @@ def send_github_notifications?
 
   def production?
```

Comment:
looks like the wrong place to add this logic ... the stage is still `production?` ... patch should be in the pipeline part